values-es/strings.xml

```diff
@@ -135,24 +135,24 @@
             <string name="label_md5_checksum">Checksum MD5</string>
             <string name="label_copy_checksum">Copiar Checksum</string>
             <string name="file_not_found">Archivo no encontrado</string>
-            <string name="label_versionlog">VersionLog</string>
+            <string name="label_versionlog">Registro de cambios</string>
             <string name="label_show_md5">Mostrar Checksum MD5</string>
-            <string name="label_show_versionlog">Ver VersionLog</string>
+            <string name="label_show_versionlog">Ver registro de cambios</string>
             <string name="label_share">Compartir</string>
-            <string name="label_new">nuevo</string>
+            <string name="label_new">Nuevo</string>
```

Comment:
This was on purpose without a capitalized letter (as in English) because it is shown behind other text